### PR TITLE
feat: export more helpers in videojs object

### PR DESCRIPTION
**Pull request by hugorogz**
## Description
- Export all of the Video.js helpers (from utils) on the Video.js object. 

## Requirements Checklist
- [ ] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [ ] Change has been verified in an actual browser (Chrome, Firefox, IE)
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](https://codepen.io/gkatsev/pen/GwZegv?editors=1000#0))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: src/js/video.js [hunk 1]
```diff
@@ -500,5 +509,36 @@ videojs.defineLazyProperty = defineLazyProperty;
...
 videojs.addLanguage('en', {'Non-Fullscreen': 'Exit Fullscreen'});
 
+const helpers = {
+  browser,
+  buffer: Buffer,
+  clamp,
+  computedStyle,
+  createLogger,
+  defineLazyProperty,
+  domData: DomData,
```
Comment (gkatsev) [inline src/js/video.js:519]: this property is specifically not exported because it's contains internal proprties

**Review by gkatsev — CHANGES_REQUESTED** [review]
I don't think that the same thing should be exported in multiple locations, it increases the API surface unnecessary.

I would also argue that some have no real utility being exported like guid.

Each item should be considered whether it should be exported, rather than exporting it because it exists.

**Review by misteroneill — CHANGES_REQUESTED** [review]
I agree, I think we need to be more deliberate with this one. But in fairness to @hugorogz, the requirements were... limited...

> Export all of the Video.js helpers (from utils) on the Video.js object. Video.js is a library and it has a lot of useful and well tested functions that we often need in other projects. I think this currently means that we duplicate the code in places and we should instead just export all utils.

**Comment by misteroneill** [comment]
After talking with @hugorogz, I went ahead and updated the PR a bit since the requirements were unclear and that's not on him to resolve! Instead of writing detailed requirements, this was the more expedient path. Thanks, Hugo!

**Review by brandonocasey — COMMENTED** [review]
I think we might also want to add `getAbsoluteURL` and `getFileExtension` from `src/js/utils/url.js` as they are not exported right now. I also wonder if we should add all of these under a `videojs.utils` or `videojs.helpers` and deprecate any helpers at the top level. imo having these on the top level object could be a bit weird for users and makes it hard to tell util functions from actual API functions.

**Comment by gkatsev** [comment]
looks like all url stuff is being exported via videojs.url.

My main worry with exposing all these utils is that it expands the API surface and now if we wanted to change/remove a helper because Video.js itself no longer uses it. That said, I don't have any major objections to this.

**Comment by misteroneill** [comment]
@brandonocasey @gkatsev What do you think about merging this as-is for now and then having a follow-up discussion about what the final exposed API surface is for utils/helpers?

**Comment by brandonocasey** [comment]
Yeah I think this pull request is good to go for now and we can talk about future API stuff another time

**Review by brandonocasey — APPROVED** [review]

**Review by gkatsev — APPROVED** [review]

**Pull request merged**